classes/Homepage.php

```diff
@@ -47,6 +47,7 @@ public function display() {
         $data['topics'] = $this->getFrontPageTopics();
         $data['divisions'] = $this->getRecentDivisions();
         $data['search_box'] = $this->getSearchBox($data);
+        $data['current_assembly'] = "scotland";
```

Comment:
I don't think this is correct as it means the uk page gets scotland as the current assembly.

Maybe make current_assembly into a property and then override that in SPHomepage?